editor/components/post-saved-state/index.js

```diff
@@ -60,10 +59,15 @@ export function PostSavedState( { isNew, isPublished, isDirty, isSaving, isSavea
 	}
 
 	return (
-		<Button className={ classnames( className, 'button-link' ) } onClick={ onSave }>
+		<IconButton
+			className="editor-post-save-draft"
+			onClick={ onSave }
+			icon="cloud-upload"
+			label={ __( 'Save Draft' ) }
```

Comment:
The behavior of `label` in this case will apply an `aria-label` to the element. I guess this should probably have no practical effect other than to defer to this label in all cases (even in mobile context), which may be desirable I suppose.

Reply:
That was sort of the thinking behind adding this. I still feel like it may not be ideal to have two elements output here, one for mobile, one for desktop, but I can't think of a better way to make a good experience for both.

I'll leave the label in and we can revisit if need be.